explorer/app/components/Links/Item.tsx

```diff
@@ -32,14 +37,9 @@ export const Item = ({
   return (
     <Box display="flex">
       <Link href={link.url} passHref>
-        <Text
-          variant="text-body-400-2lines"
-          component="a"
-          customColor="primaryAnvil"
-          sx={{ wordBreak: "break-word" }} // TODO(cc) to be resolved with #89 (Update citation component to match refined mocks)
-        >
+        <MLink variant="text-body-400-2lines">
```

Comment:
TODO - this component should be superseded by `Link` however it is still in use. Refactored `Text` to a Mui `Link` component to achieve correct link styles.

Reply:
Should we add a corresponding `// TODO` in the code here to record the required update beyond this PR?

Reply:
Sure! There are a number of components that could become obsolete. I'll make this one up.